api/base/settings/local-dist.py

```diff
@@ -0,0 +1,3 @@
+# API_PATH is '/api' on staging/production, '' on develop
+API_PATH = ''
+API_BASE = 'v2/'
```

Comment:
Just deployed this to staging and all is good w/ one issue we should add the following so the `STATIC_URL` is updated.

```python
STATIC_URL = '{}/{}static/'.format(API_PATH, API_BASE)
```